File: js/nova/star/WindowThread.js
var WindowThread = function () {
	var self = this;
	
	this.window = nova_null;
	
	this.WindowThread = function (window) {
		self.this(window);
	};
	
	this.destroy = function () {
	};
	
	this.this = function (window) {
		self = window;
	};
	
	this.run = function () {
		self.nova_createWindow(self.window.x, self.window.y, self.window.width, self.window.height, self.window.title.chars.data);
	};
	
	this.super = function () {
	};
	
	
};

File: java/nova/star/WindowThread.java
package nova.star;

import java.util.Optional;
import nova.exception.ExceptionData;
import nova.exception.Exception;
import nova.exception.DivideByZeroException;
import nova.io.NovaConsole;
import nova.primitive.number.Number;
import nova.primitive.number.NovaByte;
import nova.primitive.number.NovaShort;
import nova.primitive.number.NovaInt;
import nova.primitive.number.NovaLong;
import nova.primitive.number.NovaFloat;
import nova.primitive.number.NovaDouble;
import nova.primitive.Null;
import nova.primitive.number.Char;
import nova.primitive.Bool;
import nova.datastruct.list.NovaArray;
import nova.datastruct.list.IntArray;
import nova.datastruct.list.CharArray;
import nova.datastruct.list.DoubleArray;
import nova.datastruct.list.IntRange;
import nova.thread.Thread;
import nova.thread.async.Async;
import nova.gc.GC;
import nova.math.NovaMath;
import nova.NovaObject;
import nova.NovaString;
import nova.System;
import nova.Class;
import nova.star.Window;

public class WindowThread extends Thread
{
	private Window window;
	
	
	
	
	
	public WindowThread(Window window)
	{
		init(window);
	}
	
	
	public void init(Window window)
	{
		this.window = window;
	}
	
	public void run()
	{
		nova_createWindow(window.x, window.y, window.width, window.height, window.title.chars.data);
	}
	
}